finality-aleph/src/sync/service.rs

```diff
@@ -328,6 +350,7 @@ where
 
     fn handle_task(&mut self, task: RequestTask<BlockIdFor<J>>) {
         trace!(target: LOG_TARGET, "Handling task {}.", task);
+        self.metrics.report_sync_handle_task();
```

Comment:
this is fine I think, but maybe move to the very end for clarity?

Reply:
done